pytest-selenium/pages/content.py

```diff
@@ -31,6 +47,24 @@ def sidebar(self):
     def attribution(self):
         return self.Attribution(self)
 
+    @property
+    def section_url_within_attribution(self):
+        return self.find_element(*self._section_url_locator)
+
+    def click_next_link(self):
+        title_before_click = self.title.get_attribute("innerHTML")
+        self.offscreen_click(self.next_link)
+        self.wait.until(
+            lambda _: title_before_click != (self.title.get_attribute("innerHTML") or "")
+        )
```

Comment:
this `wait.util` on the title is neat, but maybe we should move it to a helper function so we don't have to put it in multiple places? and then if we ever update the logic for being able to tell when navigation is done it'll only be in one place?

Reply:
created a helper function